package/kernel/linux/modules/video.mk

```diff
@@ -1030,3 +1030,38 @@ define KernelPackage/video-gspca-konica/description
 endef
 
 $(eval $(call KernelPackage,video-gspca-konica))
+
+define KernelPackage/drm-i915
+  SUBMENU:=$(VIDEO_MENU)
+  TITLE:=Intel GPU drm support
+  DEPENDS:=@TARGET_x86 +kmod-drm-ttm +kmod-drm-kms-helper +i915-firmware
```

Comment:
TARGET_X86 is true for all 4 subtargets not only for the 64 bit subbtarget. 
You should also modify the other target configuration files:
````
hauke@hauke-t480:~/openwrt/openwrt$ git grep CONFIG_DRM_I915= -- target/linux/x86/
target/linux/x86/64/config-5.10:CONFIG_DRM_I915=y
target/linux/x86/64/config-5.15:CONFIG_DRM_I915=y
target/linux/x86/generic/config-5.10:CONFIG_DRM_I915=y
target/linux/x86/generic/config-5.15:CONFIG_DRM_I915=y
target/linux/x86/legacy/config-5.10:CONFIG_DRM_I915=y
target/linux/x86/legacy/config-5.15:CONFIG_DRM_I915=y
hauke@hauke-t480:~/openwrt/openwrt$ 
````